core/src/modules/middleware/authentication.ts

```diff
@@ -93,6 +87,41 @@ async function authenticateTeamMember(
     },
     { catchError: true }
   );
+}
+
+// Authenticate user from file in .local directory
+async function authenticateConfigUser(
+  data: { [key: string]: any },
+  optional: boolean
+) {
+  if (optional) return;
+  try {
+    const user = await ConfigUser.get();
+    if (user?.email !== true) {
+      const error = new Error("Config user not properly set.");
+      error["code"] = "AUTHENTICATION_ERROR";
+      throw error;
+    }
+  } catch (err) {
+    return err;
+  }
+}
+
+// Conditionally choose auth method based on run mode.
+async function authenticateTeamMember(
+  data: { [key: string]: any },
+  optional: boolean
+) {
+  let error: Error;
+
+  if (
+    process.env.GROUPAROO_RUN_MODE === "cli:config" &&
```

Comment:
👍 Can you please test that in the regular run modes you cannot authenticate with a .local user.